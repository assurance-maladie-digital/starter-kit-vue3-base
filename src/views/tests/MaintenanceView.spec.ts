import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import Maintenance from '../MaintenanceView.vue'
import { vuetify } from '../../../tests/unit/setup'

describe('MaintenanceView', () => {
    it('renders properly', () => {
        const wrapper = mount(Maintenance, {
            global: {
                plugins: [vuetify],
            },
        })
        expect(wrapper.html()).toMatchSnapshot()

        const h2Text = wrapper.find('h2').text()
        expect(h2Text).toContain('Maintenance en cours')
    })
})
