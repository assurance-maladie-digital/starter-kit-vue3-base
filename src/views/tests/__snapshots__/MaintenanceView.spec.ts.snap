// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MaintenanceView > renders properly 1`] = `
"<div data-v-813c93fa="" data-v-4104c579="" data-v-af35938b="" class="py-10 px-8 vd-page-container d-flex justify-center">
  <div data-v-813c93fa="" class="v-sheet v-theme--light bg-transparent" style="width: 800px;">
    <div data-v-4104c579="" class="v-card v-theme--light v-card--density-default elevation-2 v-card--variant-elevated pa-6 pa-sm-16">
      <!---->
      <div class="v-card__loader">
        <div class="v-progress-linear v-theme--light v-locale--is-ltr" style="top: 0px; height: 0px; --v-progress-linear-height: 2px;" role="progressbar" aria-hidden="true" aria-valuemin="0" aria-valuemax="100">
          <!---->
          <div class="v-progress-linear__background"></div>
          <div class="v-progress-linear__buffer" style="width: 0%;"></div>
          <transition-stub name="fade-transition" appear="false" persisted="false" css="true">
            <div class="v-progress-linear__indeterminate">
              <div class="v-progress-linear__indeterminate long"></div>
              <div class="v-progress-linear__indeterminate short"></div>
            </div>
          </transition-stub>
          <!---->
        </div>
      </div>
      <!---->
      <!---->
      <div data-v-4104c579="" class="v-row max-width-none">
        <div data-v-4104c579="" class="v-col-sm-6 v-col-12 order-last order-sm-first text-center text-sm-left">
          <div data-v-4104c579="" class="vd-code font-weight-thin text-primary mb-4"><span data-v-4104c579="" class="d-sr-only">Code d'erreur&nbsp;: </span> </div>
          <h2 data-v-4104c579="" class="mb-2 font-weight-bold text-h5 mb-4">Maintenance en cours</h2>
          <p data-v-4104c579="">L’application n’est pas disponible pour le moment, veuillez nous excuser pour la gêne occasionnée.</p>
          <!---->
        </div>
        <div data-v-4104c579="" class="v-col-sm-6 v-col-12 d-flex align-center justify-center"><img data-v-af35938b="" src="data:image/svg+xml,%3csvg%20xmlns='http://www.w3.org/2000/svg'%20fill='none'%20viewBox='0%200%20171%20172'%3e%3cpath%20fill='%23e7ecf5'%20fill-rule='evenodd'%20d='M170.98%2085.88c-.67%2023.17-21.93%2038.99-39.79%2053.75-15.66%2012.94-33.5%2021.47-53.71%2023.52-23.03%202.34-49.9%205.4-65.83-11.4C-4.15%20135.07-.9%20108.28%204.2%2085.88%208.22%2068.22%2023.74%2057.72%2036.11%2044.5c13.13-14.02%2022.4-33.9%2041.37-36.9%2020.81-3.3%2040.79%207.79%2056.95%2021.32%2017.98%2015.05%2037.24%2033.52%2036.55%2056.97Z'%20clip-rule='evenodd'/%3e%3ccircle%20cx='123.87'%20cy='127.71'%20r='17.83'%20fill='%23fff'%20stroke='%230c419a'%20stroke-width='2'/%3e%3ccircle%20cx='126.62'%20cy='126.25'%20r='15.08'%20fill='%23fff'%20stroke='%230c419a'%20stroke-width='2'/%3e%3ccircle%20cx='126.95'%20cy='125.92'%20r='10.86'%20fill='%23fff'%20stroke='%230c419a'%20stroke-width='2'/%3e%3cpath%20fill='%230c419a'%20d='M87.88%2067.72c0-19.13%2013.03-28.19%2024.48-30.48%2021.07-4.21%2054.13%2012.81%2054.63%2057.07.42%2036.27-15.75%2076.47-51.55%2076.52-32.75.05-41.02-27.72-45.88-34.2-1.66-2.2-5.02-8.76-8.76-11.68-2.48-1.93-8.1-1.9-12.48-3.73-4.8-2-11.98-8.02-13.46-13.94l-5.19-22.54c2.4-1.93%203.9-2.06%206.65-1.78.97%202.54%203.37%2010.22%206.49%2020.6%203.74%2012.47%2013.61%2011.83%2022.37%209.4%208.75-2.44%2010.7-13.46%209.73-19.46-.78-4.8-5.73-15.19-8.11-19.78%202.02-2.3%203.46-2.8%206.32-2.92%201.2%202.43%204.22%208.66%206.81%2014.1%203.24%206.82%203.73%2012.81%203.24%2016.87-.48%204.05-4.37%2012.32-6.32%2013.62-2.43%201.78-1.93%203.73-2.1%206.8-.36%206.15%208.87%2041.35%2040.69%2041.19%2034.77-.19%2045.7-47.56%2045.4-69.07-.6-41.99-30.97-55.12-46.54-51.07-8.47%202.2-19.74%206.98-20.26%2024.48-.46%2015.12%2012.9%2031.98%2021.23%2041.52%201.42%201.63%202.7%203.04%203.73%204.2-.64%202.43-2.75%204.05-4.7%204.86a217.2%20217.2%200%200%201-4.05-4.34c-12.78-14.09-22.37-28.9-22.37-46.24Z'/%3e%3cpath%20fill='%230c419a'%20d='M120.8%2035.67c-3-.16-5.91.05-8.64.59a32.39%2032.39%200%200%200-17.06%209.22c-4.85%205.12-8.22%2012.45-8.22%2022.24%200%2017.7%209.78%2032.75%2022.63%2046.91l.74-.67-.74.67c1.34%201.47%202.7%202.93%204.07%204.36l.47.5.63-.27c2.16-.88%204.54-2.68%205.29-5.52l.13-.52-.35-.4c-1.04-1.17-2.3-2.57-3.72-4.2l-.75.66.74-.66c-8.3-9.5-21.42-26.32-20.98-40.83.25-8.54%203.08-13.8%206.83-17.28%203.75-3.48%208.51-5.18%2012.68-6.26%2014.75-3.84%2044.7%208.6%2045.29%2050.11.15%2010.58-2.5%2027.72-9.45%2042.08-6.95%2014.37-18.02%2025.89-34.95%2025.98-15.55.08-25.46-8.41-31.57-17.78a56.35%2056.35%200%200%201-6.43-13.66c-1.3-4.08-1.76-7.46-1.7-8.7.1-1.6.03-2.84.2-3.75.16-.92.45-1.52%201.5-2.3l-.04.03a7.72%207.72%200%200%200%202.04-2.4%2034.63%2034.63%200%200%200%202.03-3.79c1.3-2.8%202.42-5.88%202.7-8.14.5-4.26-.02-10.43-3.34-17.42-2.59-5.44-5.6-11.67-6.81-14.11l-.3-.59-.64.03c-2.96.13-4.9.84-7.03%203.26l-.45.51.31.61c1.18%202.27%203%206%204.63%209.75%201.62%203.74%203.03%207.55%203.38%209.73.45%202.8.22%206.93-1.16%2010.55-1.39%203.6-3.83%206.66-7.85%207.78-4.3%201.2-8.8%201.9-12.54.9-3.73-1.02-6.8-3.62-8.61-9.63-2.06-6.86-3.06-9.84-4.3-13.66l-2.19-6.94-.02-.06-.2-.56-.63-.08c-2.83-.28-4.84-.04-7.38%202l-.49.4%205.35%2023.15v.01c.8%203.25%203.09%206.34%205.75%208.92a28.5%2028.5%200%200%200%208.28%205.7c2.34.98%204.9%201.43%207.14%201.9%202.25.45%204.17.96%205.12%201.7%201.7%201.32%203.44%203.63%204.94%205.87%201.49%202.24%202.71%204.4%203.63%205.61%201.1%201.46%202.56%204.44%204.54%207.98a67.85%2067.85%200%200%200%207.99%2011.64c6.9%207.9%2017.5%2015.02%2034.15%2014.99%2018.28-.03%2031.63-10.37%2040.22-25.18%208.59-14.8%2012.54-34.06%2012.33-52.35-.44-39.16-26.18-57.56-47.2-58.63Zm-.12%202c19.88%201.01%2044.88%2018.36%2045.31%2056.65.2%2017.98-3.72%2036.95-12.06%2051.33-8.34%2014.38-20.96%2024.16-38.5%2024.18-16.08.02-26-6.72-32.64-14.31a65.94%2065.94%200%200%201-7.74-11.3c-1.96-3.49-3.35-6.41-4.69-8.2-.74-.98-2.03-3.2-3.57-5.52-1.53-2.3-3.33-4.74-5.37-6.34-1.54-1.2-3.67-1.62-5.95-2.09-2.28-.46-4.72-.91-6.76-1.77a27%2027%200%200%201-7.67-5.3c-2.5-2.41-4.54-5.3-5.2-7.96l-5.03-21.8a6.11%206.11%200%200%201%204.78-1.28l6.26%2019.88c1.94%206.47%205.67%209.8%2010%2010.97%204.33%201.18%209.15.34%2013.6-.9%204.73-1.31%207.64-4.98%209.17-8.99%201.54-4%201.8-8.38%201.28-11.58-.43-2.62-1.88-6.4-3.53-10.2a160.61%20160.61%200%200%200-4.34-9.16%205.98%205.98%200%200%201%204.51-2.08c1.3%202.64%204.02%208.25%206.49%2013.43%203.16%206.65%203.61%2012.46%203.15%2016.32-.21%201.8-1.28%204.87-2.53%207.54a32.87%2032.87%200%200%201-1.9%203.56%208.1%208.1%200%200%201-1.46%201.8l-.01.02-.02.01a5.54%205.54%200%200%200-2.3%203.55c-.22%201.27-.13%202.53-.2%204-.12%201.85.43%205.18%201.77%209.41a58.24%2058.24%200%200%200%206.66%2014.16c6.36%209.73%2016.98%2018.76%2033.26%2018.68%2017.83-.1%2029.6-12.36%2036.74-27.1%207.13-14.75%209.8-32.05%209.65-42.98-.6-42.47-31.4-56.3-47.8-52.03-4.3%201.12-9.4%202.9-13.53%206.74-4.13%203.83-7.2%209.72-7.47%2018.68-.47%2015.41%2012.52%2031.85%2020.92%2041.53a20.25%2020.25%200%200%201-3.57%203.35c-12.38-13.8-21.5-28.16-21.5-44.85%200-9.34%203.14-16.1%207.66-20.86a30.46%2030.46%200%200%201%2016-8.64c2.55-.5%205.29-.7%208.13-.55zm-5.41%2073.07%202.52%202.84a6.6%206.6%200%200%201-3.32%203.44c-.92-.97-1.84-1.96-2.75-2.95%201.3-.98%202.49-2.1%203.55-3.33z'%20color='%23000'/%3e%3cpath%20fill='%23fff'%20stroke='%230c419a'%20stroke-width='2'%20d='m34.44%2082.53.1.36a6.74%206.74%200%200%200-3.73%201.04C26.1%2064.48%2015.89%2025.9%2028.38%2012.6c4.54-4.54%2010.21-6.1%2012.48-6.32l.81%203.56a22.53%2022.53%200%200%200-12%206.81c-9.96%2010.61%201.1%2052.13%204.77%2065.88Z'/%3e%3cpath%20fill='%23fff'%20stroke='%230c419a'%20stroke-width='2'%20d='M37.94%2018.27c18%2011.03%2028.86%2042.16%2033.4%2052.7A6.49%206.49%200%200%200%2068%2072.59c-6.39-16.54-19.2-44.02-30.06-50.75C29.25%2016.46%2021.9%2018.6%2018%2020.71c-.05-1.7-.35-2.44-1.3-3.4%206.65-2.44%2014.1-3.74%2021.24.96Z'/%3e%3cpath%20fill='%230c419a'%20stroke='%230c419a'%20stroke-width='2'%20d='M46.86%2010c-2.47%200-4.7%201.3-5.84-1.61-.65-1.79.16-3.41%201.46-3.9%201.3-.48%202.43-.64%202.92-1.13a7.2%207.2%200%200%201%203.73-2.1c2.24-.38%204.54.48%205.51%202.91a4.28%204.28%200%200%201-2.76%205.84c-1.45.48-3.72%200-5.02%200Z'/%3e%3cellipse%20cx='52.51'%20cy='5.35'%20fill='%23fff'%20stroke='%23fff'%20stroke-width='2'%20rx='1.12'%20ry='2.28'%20transform='rotate(-13%2052.51%205.35)'/%3e%3cpath%20fill='%230c419a'%20stroke='%230c419a'%20stroke-width='2'%20d='M10.7%2017.3c2.47%200%205.74-1.94%206.98%201.46.64%201.78.11%203.7-1.18%204.18-1.3.48-2.44.65-2.92%201.13a7.2%207.2%200%200%201-3.73%202.11c-2.25.38-4.67-.44-5.51-2.92-.77-2.23-.26-4.37%201.83-5.47%201.78-.94%202.75-.49%204.53-.49Z'/%3e%3cellipse%20cx='6.47'%20cy='22.53'%20fill='%23fff'%20stroke='%23fff'%20stroke-width='2'%20rx='1.12'%20ry='2.28'%20transform='rotate(158%206.47%2022.53)'/%3e%3c/svg%3e" alt="Maintenance en cours"></div>
      </div>
      <!---->
      <!----><span class="v-card__underlay"></span>
    </div>
  </div>
</div>"
`;
